roles/openshift_control_plane/tasks/upgrade.yml

```diff
@@ -113,19 +115,5 @@
     value: "{{ runtime_config.result | join(',') | regex_replace('(?:,)*apis/settings\\.k8s\\.io/v1alpha1=true','') }}"
   when: runtime_config.result
 
-- name: Remove old service information
```

Comment:
We do need to remove at least `/usr/lib/systemd/system/` services

Reply:
yes, I will revert that change.